Nodejs/Setup/swix/Targets_files.swr

```diff
@@ -0,0 +1,9 @@
+use vs
+
+package name="Microsoft.VisualStudio.NodejsTools.Targets"
+    version=1.2
+
+folder "InstallFolder:\"
+    folder "MSBuild\Microsoft\VisualStudio\v15.0\Node.js Tools"
+        file source=$(BuildOutputRoot)\Binaries\Nodejs\Microsoft.NodejsTools.targets
+        file source=$(BuildOutputRoot)\Binaries\Nodejs\Debugger\RemoteDebug\Microsoft.NodejsTools.WebRole.dll
```

Comment:
same with Microsoft.NodejsTools.WebRole.dll - both should land in Node.js Tools vsix - "Node.js Tools for Visual Studio\1.2\RemoteDebug\*"